guides/common/modules/proc_configuring-ipxe-environment.adoc

```diff
@@ -0,0 +1,55 @@
+[id="Configuring_iPXE_Environment_{context}"]
+= Configuring iPXE environment
+
+Configure iPXE environment on all {SmartProxies} that you want to use for iPXE provisioning.
```

Comment:
```suggestion
Configure an iPXE environment on all {SmartProxies} that you want to use for iPXE provisioning.
```

Reply:
The latter is not appropriate, because SmartProxies are the instrument to provision hosts. I will add the article.